account_financial_report_block_pdf/wizard/aged_partner_balance_wizard.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, models
+from odoo.exceptions import ValidationError
+
+
+class AgedPartnerBalanceWizard(models.TransientModel):
+    _inherit = 'aged.partner.balance.wizard'
+
+    @api.model
```

Comment:
I think that you don't need to duplicate this function in all the models.

All models inherit from se absctract one, so you should be able to only have the function in the abstract_wizard.py file.

If not, i think that the abstract_wizard.py  file should be deleted

https://github.com/OCA/account-financial-reporting/blob/12.0/account_financial_report/wizard/aged_partner_balance_wizard.py